andela/wg-rogue-one
# Update muscle and exercise lists

### What does this PR do?
There are already lists of muscles and exercises in the system, so this PR updates or adds more data to the existing data.

### Description of Task to be completed?
Update list of available muscles as well as the exercises.

### How should this be manually tested?
1. Follow the setup instructions in [README.rst](https://github.com/andela/wg-rogue-one/blob/ft-update-muscle-list-157731621/README.rst)
2. Run the ```python manage.py runserver```  command to start the app.
3. Log in as an admin with default username: **admin** and default password:  **admin**.
4. From the dashboard, click on the Exercises tab, from the drop down menu click on the muscle overview link.
5. From the overview page you should be able to see list of muscles with the new added muscles, hover over the muscles to see the list of exercises added to them. 

### Screen Shots

Page showing Muscle1 overview and list of exercises

<img width="1280" alt="muscle1" src="https://user-images.githubusercontent.com/12033494/41617076-458f5ef2-7408-11e8-8a10-4cb62ea8f7ad.png">



Page showing Muscle2 overview and list of exercises

<img width="1280" alt="muscle2" src="https://user-images.githubusercontent.com/12033494/41617153-68d70ff4-7408-11e8-9dec-4dffaa2e23e4.png">




### Any background context you want to provide?
New muscles added are:
1. Lateral deltoid 
2. Posterior Deltoid 
3. Forearms 
4. Erector spinae. This muscle already existed in the muscle lists but didn't have an SVG image and exercises. So I added an SVG image and lists of exercises to it. 

### What are the relevant pivotal tracker stories?

[#157731621](https://www.pivotaltracker.com/story/show/157731621)
